### PR TITLE
Create AiplaneValidator, implement Prediacate and Pageable in service

### PR DESCRIPTION
**Pull request by blinii**

### CODE REVIEW

File: src/main/java/com/devfreaks/tripper/repositories/AirplaneRepository.java [hunk 1]
```diff
@@ -1,9 +1,24 @@
...
 import org.springframework.data.repository.CrudRepository;
 import org.springframework.stereotype.Repository;
 
+import java.util.UUID;
+
 @Repository
-public interface AirplaneRepository extends CrudRepository<Airplane, String> {
-}
+public interface AirplaneRepository extends PagingAndSortingRepository<Airplane, UUID>,
+        QueryDslPredicateExecutor<Airplane>, QuerydslBinderCustomizer<QAirline> {
```
Comment (flakron) [inline src/main/java/com/devfreaks/tripper/repositories/AirplaneRepository.java]: ja ke huq klases cilen e ke perdor
```QAirline``` -> ```QAirplane```

File: src/main/java/com/devfreaks/tripper/repositories/AirplaneRepository.java [hunk 2]
```diff
@@ -1,9 +1,24 @@
...
+
+    default void customize(QuerydslBindings bindings, QAirline airplane) {
```
Comment (flakron) [inline src/main/java/com/devfreaks/tripper/repositories/AirplaneRepository.java]: njejt edhe ktu

File: src/main/java/com/devfreaks/tripper/repositories/AirplaneRepository.java (hunk 1 above)
Comment (blinii) [inline src/main/java/com/devfreaks/tripper/repositories/AirplaneRepository.java]: op

File: src/main/java/com/devfreaks/tripper/repositories/AirplaneRepository.java (hunk 2 above)
Comment (blinii) [inline src/main/java/com/devfreaks/tripper/repositories/AirplaneRepository.java]: shum t'peraferta bre nonen

**Pull request merged**